shelley/chain-and-ledger/formal-spec/epoch.tex

```diff
@@ -548,6 +556,13 @@ \subsection{Pool Reaping Transition}
       \trans{poolreap}{e}
       \left(
         \begin{array}{rcl}
+          \var{utxo} \\
+          \varUpdate{\var{deposits}}
+          & \varUpdate{-}
+          & \varUpdate{(\var{unclaimed} + \var{refunded})} \\
```

Comment:
I suppose this cannot be negative in total? Maybe add a not on that property.

Reply:
yes, that's the property, I'll add it to the list I've got going!